### DISCUSSION TITLE
Discord request: screenshot channel (not competition)

### DISCUSSION THREAD

**Discussion by Turtle-PL** [OpenTTD/OpenTTD#9354] [Features]
Hi,
I've did notice the lack of screenshot channel in OpenTTD's Discord.
I think it would be nice sharing your screenshots from the game. 
It may sound stupid, but I think that would actually make sense, because I've seen many people send screenshots from the game in the #openttd chat and then vanishing into nowhere. 
I don't think I am the only  person that would think that channel would be actually used and liked.
Open for opinions!

**Comment by glx22** [comment]
I think this should be discussed on Discord, not here.

**Comment by Shump** [comment]
There is now `#openttd-screenshots`: https://discord.com/channels/142724111502802944/853284744056799252
